src/code.c

```diff
@@ -2257,12 +2257,7 @@ void CodeAssLVar (
     Expr                rhsx;           /* right hand side expression      */
 
     /* allocate the assignment                                             */
-    if ( lvar <= 16 ) {
-        ass = NewStat( T_ASS_LVAR + lvar, 2 * sizeof(Stat) );
-    }
-    else {
-        ass = NewStat( T_ASS_LVAR,        2 * sizeof(Stat) );
-    }
```

Comment:
Hmmm... so `T_ASS_LVAR_xx` is/are used after all -- apparently as an optimization, and this PR removes the optimization.

So, did you perform any benchmarking to see how this affects things?